### ISSUE TITLE
Hide hidden categories for DMs

### ISSUE THREAD

**Issue by nivthefox** [p4535992/inventory-plus#23]
As a DM, it's confusing to me to look at a player's sheet and see the hidden categories.  It'd be nice if there was an option to "hide" these categories from my view, as well, unless the sheet were opened for editing.

**Comment by p4535992** [comment]
@nivthefox  Sorry what did you mean with "hidden categories" the default categories from dnd5e system ?

**Comment by nivthefox** [comment]
Yes, the default categories.  Right now, if I choose to hide--say--the "loot" category (which is a default), it will still be visible to me as a DM, but not to the players who own the sheet.
